supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -329,3 +329,12 @@ See xref:release-notes[Release notes] for guidelines about deprecation notices.
 image:images/no.png[no] We will fix this issue in the 18.3.4 release next February.
 
 image:images/yes.png[yes] It is anticipated that an upcoming release will include a fix for this issue.
+
+[[references-to-cost]]
+== References to cost
+
+Avoid all references to the costs and charges of Red Hat products. While the IBM style guide blocks the usage of `free`, these references should also be avoided since they can confuse users and cause legal concerns. Any cost information is best left to marketing materials.
+
+.Example phrases
+
+* "at no initial cost" - This phrase was removed from the documentation because while it implies there are further costs, it can also be construed to mean that the product is free when it is not.
```

Comment:
I know what you're talking about here - that it was removed during peer review. But other readers might not. Suggest rewording a bit (and adding another comma)

```suggestion
* "at no initial cost" - This phrase should be avoided in documentation because, while it implies there are further costs, it can also be construed to mean that the product is free when it is not.
```